charts/airflow-operator/templates/crds/airflow.yaml

```
@@ -0,0 +1,3356 @@
+{{ if .Values.crd.create }}
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: '{{ .Release.Namespace }}/airflow-operator-serving-cert'
+  name: airflows.airflow.apache.org   ## Cannot have release-name added since CustomResourceDefinition.apiextensions.k8s.io "astronomer-airflows.airflow.apache.org" is invalid: metadata.name: Invalid value: "astronomer-airflows.airflow.apache.org": must be spec.names.plural+"."+spec.group
```

nit: i think we could improve this comment.
```suggestion
  # name must be spec.names.plural+"."+spec.group https://github.com/kubernetes/kubernetes/blob/b4f902f037/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L80
  name: airflows.airflow.apache.org
```